azure_monitor/azfunc_sample/README.md

```diff
@@ -1,90 +1,78 @@
+az login
 
-> *Important. Once the issues (see below) are resolved in the platform, it is recommended to move to updated OpenCensus Azure Monitor Log Exporter implementation!* 
+func azure functionapp publish python-appinsights-SUFFIX   
+```
 
-## Run Locally
+- You should see the two function applications deployed successfully:
 
-### Setup
+  ![The results of the function app publish command is displayed.](../media/python_functionapp_deploy.png "Review the results and ensure the function apps deployed succesfully.")
 
-1. Install [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=linux%2Ccsharp%2Cbash).
-1. Install Azure Storage Emulator [Azurite](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite) & start Azurite's Blob service 
-1. Install all package dependencies locally using `pip install -r requirements.txt`.
-1. Create a file named `local.settings.json` which contains the following configuration information (*replace the placeholders*)
+- Switch to the Azure Portal.
+- Navigate to the **python-appinsights-SUFFIX** function app.
+- Under **Functions**, select **Functions**, you should see the two python function apps deployed.
 
-    ```json
-    {
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-        "FUNCTIONS_WORKER_RUNTIME": "python",
-        "PYTHON_ENABLE_WORKER_EXTENSIONS": "1",
-        "APPLICATIONINSIGHTS_CONNECTION_STRING": "InstrumentationKey=[YOUR-APPLICATION-INSIGHTS-KEY]",
-        "WEBSITE_SITE_NAME": "[YOUR-APPLICATION-INSIGHTS-CLOUD-ROLE-NAME e.g. MyFunction]",
-        "EXTERNAL_DEPENDENCY_URL":"[SOME-WEB-URL-FOR-DEPENDENCY-TRACKING e.g. https://www.bing.com]"
-        }
-    }
-    ```
+> **NOTE** Azure Function runtimes support various Python versions depending on the version selected.  See [Languages by runtime version](https://docs.microsoft.com/en-us/azure/azure-functions/supported-languages#languages-by-runtime-version).
 
-### Test
+- Review the function app code.  Notice the usage of the a storage account and a call to the database.
 
-1. (Optional) Update environment variables in file `.test.env`
-1. Run tests with `pytest`
+## 4 - Review Dependency Log Data
 
-### Run
+- Navigate to the **python-appinsights-SUFFIX** Application Insights resource, under **Monitoring**, select **Logs**
+- Run the following query:
 
-1. Start the Azure Function host: `func host start`
-1. Send HTTP request. `curl http://localhost:[PORT]/api/instrumentation`
+```kusto
+dependencies
+```
 
-## Contributors
-[Oliver Lintner](https://github.com/se02035)  
\ No newline at end of file
+- You should see the dependencies displayed for the calls to MySQL and Azure storage displayed.
```

Comment:
Maybe include an image of the telemetry that is expected?

Reply:
Added.